frontend/app.py

```diff
@@ -1,22 +1,32 @@
 import os
-
-
-from dotenv import load_dotenv, find_dotenv
 import requests
 import streamlit as st
+from dotenv import load_dotenv, find_dotenv
+
+from pages.project_issues_chat import project_issues_chat
+from pages.issue_summary import issue_summary
+
+from components.sidebar import sidebar
+
+st.set_page_config(layout='wide', initial_sidebar_state='expanded')
 
 
 def launch_app():
-    print("launching app")
-    st.title("Github Issues PM Assistant")
+    st.title('Github PM Assistant')
+
+    status()
+
+    sidebar()
+
 
+def status():
     with st.spinner("Searching for an answer..."):
```

Comment:
Searching for an answer.
```suggestion
    with st.spinner("Checking if API reachable..."):
```